pkg/dnshelper/dnshelper.go

```diff
@@ -0,0 +1,46 @@
+package dnshelper
+
+import (
+	"fmt"
+	"html/template"
+
+	"github.com/massalabs/station/pkg/convert"
+	"github.com/massalabs/station/pkg/node"
+)
+
+const (
+	metaKey = "META"
+
+	// Indexes of data in website name key.
+	indexOfWebsiteAddress     = 0 // Index of website Address in the dnsValue array
+	indexOfWebsiteDescription = 2 // Index of website Description in the dnsValue array
+)
+
+// Helper function to fetch website values from DNS entry.
+func GetWebsiteValues(dnsValue []byte) (string, string, error) {
+	websiteValue := convert.ByteToStringArray(dnsValue)
+	if len(websiteValue) <= indexOfWebsiteAddress {
+		return "", "", fmt.Errorf("invalid website value: missing website address")
+	}
+
+	websiteStorerAddress := websiteValue[indexOfWebsiteAddress]
+
+	websiteDescription := ""
+	if len(websiteValue) > indexOfWebsiteDescription {
+		websiteDescription = websiteValue[indexOfWebsiteDescription]
+	}
+	// Prevent XSS by escaping special characters in websiteDescription
+	escapedDescription := template.HTMLEscapeString(websiteDescription)
```

Comment:
this peace of code is not optimal: if `len(websiteValue) > indexOfWebsiteDescription` is false, then we will call `HTMLEscapeString` for nothing

Reply:
I would suggest to call `HTMLEscapeString` inside the if statement